lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -258,9 +263,15 @@ getTransaction = label "getTransaction" $ do
 
 putSignedTx :: (Tx, [TxWitness]) -> Put
 putSignedTx (tx@(Tx inputs outputs), witnesses) = withSizeHeader16be $ do
+    unless (length inputs < 256) $
+        fail "number of inputs must be lower than 256"
+    unless (length outputs < 256) $
+        fail "number of outputs must be lower than 256"
```

Comment:
Instead of magic number, it'd be better to use `maxBound @Word8` to show where this `256` comes from.

Reply:
Done:
```haskell
    unless (length inputs <= fromIntegral (maxBound :: Word8)) $
        fail "number of inputs cannot be greater than 255"

```
- `maxBound :: Word8 ` is `255` so I've used `255]` instead of `256)` as bound
- Preserved the 255 in the message despite the risk of going out of sync
- Avoided to state the entire allowed range, e.g. `number of inputs must be within the range [0,255]`, since I believe the 0 is a grey-zone.